Tests/Tests/STPFormEncoderTest.m

```diff
@@ -184,6 +183,15 @@ - (void)testFormEncoding_nilRootObject {
     XCTAssertEqualObjects([self encodeObject:testObject], @"test_property=success");
 }
 
+- (void)testQueryStringWithBadFields {
+    NSDictionary *params = @{
+                             @"foo]": @"bar",
+                             @"baz": @"qux[",
```

Comment:
Any chance you could add a `;` to this bad field test (it should encode to `%3B`)? We tripped over that in dashboard-ios#862